lib/driver.js

```diff
@@ -137,7 +142,9 @@ class YouiDriver extends BaseDriver {
       browserName: "",
       app: this.caps.app,
       platformVersion: opts.platformVersion,
-      deviceName: opts.deviceName
+      deviceName: opts.deviceName,
+      // Disabling the proxy CommandTimeout
+      newCommandTimeout: 0
     };
```

Comment:
(Info for Heloise) What happened was the newCommandTimeout would be started and set through the youi-driver base-driver inheritance; then we'd create an ios-driver to proxy through which was also using the base-driver and starting a default 60 second command timeout. So we're telling the proxy/ios/basedriver to never time out because we'll handle it at the youi-driver/basedriver level :)